Resolves #542 - Labeling order for two curb section of onboarding

maddalihanumateja opened the pull request:
Resolves #542. I've modified the onboarding states to have 2 branches in the storyline when we are at the 2 curb section. A branch will be triggered when the corresponding curb ramp is labelled. I've done the following changes:
1. Created a new onboarding state called "label-both-curbs" whose properties field is an array of the canvas regions that will trigger events instead of a single region. The transition field is also an array. There is a one-to-one mapping by index between the properties and transition field arrays. So executing the properties[i] event will result in going to a state dictated by the transition[i] function.
2. Created a new function called _visitLabelMultipleAccessibilityAttributeState that handles a state that has multiple possible labeling regions. Each region will lead to a different branch in the tutorial story. This isn't a general purpose function right now that handles events other than label placement. I don't think there is a necessity right now.

manaswis commented on public/javascripts/SVLabel/src/SVLabel/onboarding/Onboarding.js:
```diff
@@ -432,23 +432,32 @@ function Onboarding (svl, actionStack, audioEffect, compass, form, handAnimation
-                _visitWalkTowards(state, annotationListener);
-            } else if (state.properties.action == "Instruction") {
-                _visitInstruction(state, annotationListener);
+            if(state.properties.constructor == Array){
```
Could you update this to check for action? That way the code will be consistent

manaswis commented on public/javascripts/SVLabel/src/SVLabel/onboarding/Onboarding.js:
```diff
@@ -726,6 +735,45 @@ function Onboarding (svl, actionStack, audioEffect, compass, form, handAnimation
+                var imageY = st.properties[i].imageY;
+                var tolerance = st.properties[i].tolerance;
+
+                var clickCoordinate = mouseposition(e, this),
```
This code seems to be repeated here from _visitLabelAccessibilityAttributeState. Could you create a common function and use it it both places.

maddalihanumateja commented:
Made the following changes:
- Merged with new changes in develop
- Corrected the progress percentage for the new onboarding states.
- Added an extra if condition to check if the properties array contains labeling events for code consistency.
- All "label attribute" states have array "properties" and "transition" fields.
- Didnt modify the other states' fields since we need to rewrite their handlers. May cause unexpected bugs.
- _visitLabelMultipleAccessibilityAttributeState handles all "label attribute" states. Reduces code redundancy.

manaswis commented:
![image](https://user-images.githubusercontent.com/2873216/27099804-1f4f723c-504a-11e7-9cde-94cf8954fa9d.png)

The arrows are blinking too fast in this screen. This is the screen that shows up when the intersection corner is first shown.

maddalihanumateja commented:
I've solved the blinking frequency problem.
